src/SprykerEco/Zed/Computop/Business/Payment/Handler/Saver/Init/PayPalExpressResponseSaver.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop\Business\Payment\Handler\Saver\Init;
+
+use Generated\Shared\Transfer\ComputopPayPalExpressInitResponseTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+
+class PayPalExpressResponseSaver extends AbstractResponseSaver
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    public function save(QuoteTransfer $quoteTransfer): QuoteTransfer
+    {
+        $responseTransfer = $quoteTransfer->getPayment()->getComputopPayPalExpress()->getPayPalExpressInitResponse();
+        $this->setPaymentEntity($responseTransfer->getHeader()->getTransId());
+        if ($responseTransfer->getHeader()->getIsSuccess()) {
+            $this->getTransactionHandler()->handleTransaction(
+                function () use ($responseTransfer): void {
+                    $this->savePaymentComputopEntity($responseTransfer);
+                    $this->savePaymentComputopDetailEntity($responseTransfer);
+                    $this->savePaymentComputopOrderItemsEntities();
+                }
+            );
+        }
+
+        return $quoteTransfer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\ComputopPayPalExpressInitResponseTransfer $responseTransfer
+     *
+     * @return void
+     */
+    protected function savePaymentComputopEntity(ComputopPayPalExpressInitResponseTransfer $responseTransfer): void
```

Comment:
same issue with propel entities inside business level
this should not happen here